app/views/manageAgent/areYouSureMultipleYears.scala.html

```diff
@@ -19,16 +19,27 @@
 @this(govukErrorSummary : GovukErrorSummary, govukDetails : GovukDetails, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout, formWithCSRF: FormWithCSRF,
     govukInsetText: GovukInsetText)
 
-@(agentName: String, agentCode: Long, backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(agentName: String, agentCode: Long, backLink: String, listYears: Option[Seq[String]])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@ratingList(tag: String) = @{
```

Comment:
Should this page also have some text saying what they can't do?

E.g.
"They will not be able to see valuations on the 2023 rating list, or act on them for you." if they select 2017 and 2026?